src/rpc/blockchain.cpp

```diff
@@ -1640,6 +1640,37 @@ static T CalculateTruncatedMedian(std::vector<T>& scores)
     }
 }
 
+static constexpr int NUM_GETBLOCKSTATS_PERCENTILES = 5;
+
+template<typename T>
```

Comment:
nit: using a template here is over-engineering this. Even if you plan to re-use this a measure other than fee-rate, it would make more sense to define this just for fee-rate now and then change to a template in the next PR.